### DISCUSSION TITLE
join_with

### DISCUSSION THREAD

**Discussion by goto40** [tcbrindle/flux#196] [Q&A] [answered]
Is there something like https://en.cppreference.com/w/cpp/ranges/join_with_view in flux? I did not find it...

**Comment by tcbrindle** [comment]
Not a the moment, although there definitely should be. This is bug #124.

**Comment by tcbrindle** [comment, marked as answer]
This has now been added under the name [`flatten_with()`](https://tristanbrindle.com/flux/reference/adaptors.html#flatten-with)
